CHANGELOG.md

```diff
@@ -18,6 +18,86 @@ and includes an additional section for migration notes.
 
 ### Added
 - *ORCA-480* Added `storageClass` to Orca catalog and associated [reporting API](https://nasa.github.io/cumulus-orca/docs/developer/api/orca-api#catalog-reporting-api). Existing entries will be reported as in the `GLACIER` storage class.
+- *ORCA-479*
+    Added variable `orca_default_storage_class` which denotes the default [storage class](https://aws.amazon.com/s3/storage-classes/) to use when storing files in Orca.
+    Currently allowed values are `GLACIER` and `DEEP_ARCHIVE`
+    copy_to_glacier accepts `orcaDefaultStorageClassOverride` which can be used on a per-collection basis. If desired, add `"orcaDefaultStorageClassOverride": "{$.meta.collection.meta.orcaDefaultStorageClassOverride}` to the workflow's task's task_config.
+
+### Migration Notes
+
+- The user should update their `orca.tf`, `variables.tf` and `terraform.tfvars` files with new variables. The following optional variables have been added:
+  - orca_default_storage_class
+  
+- If desired, update collection configurations with the new optional key `orcaDefaultStorageClassOverride` that can be added to override the default S3 glacier recovery type as shown below.
+
+  ```json
+    "meta": {
+      "orcaDefaultStorageClassOverride": "Standard"
```

Comment:
I think "Standard" should be "GLACIER" or "DEEP_ARCHIVE"

Reply:
Whoops. Fixed.